src/main/java/com/google/cloud/spanner/pgadapter/wireprotocol/DescribeMessage.java

```diff
@@ -87,13 +87,21 @@ public void handleDescribePortal() throws Exception {
     if (this.statement.hasException()) {
       this.handleError(this.statement.getException());
     } else {
-      new RowDescriptionResponse(
-              this.outputStream,
-              this.statement,
-              ((DescribePortalMetadata) this.statement.describe()).getMetadata(),
-              this.connection.getServer().getOptions(),
-              QueryMode.EXTENDED)
-          .send();
+      switch (this.statement.getResultType()) {
+        case UPDATE_COUNT:
+        case NO_RESULT:
+          new NoDataResponse(this.outputStream).send();
```

Comment:
This should return a `NoDataResponse` when a row description is requested for statements that return an update count (DML) or no result (DDL). This would previously return an empty `RowDescriptionResponse` for DML/DDL statements, which again would cause some clients (native PG JDBC driver) to return an error as it does not expect an empty `RowDescriptionResponse` to be returned for DML/DDL.